randovania/games/am2r/json_data/header.txt

```diff
@@ -324,7 +324,7 @@ Dock Weaknesses
           Can Use Power Bombs
 
 
-  * Water Turbine Door
+  * Hydro Station Water Turbine
       Open:
           Trivial
```

Comment:
Don't the the turbines require missiles/bombs?

Reply:
It's trivial to open. It's lock however, is not. 
Scroll down a little bit further, and you'll see:
``` 
Lock type: FRONT_BLAST_BACK_FREE_UNLOCK
Destroy Turbine
```